storm-client/src/jvm/org/apache/storm/daemon/worker/Worker.java

```diff
@@ -246,7 +248,19 @@ public void start() throws Exception {
                             }
                         }
                     });
-              
+
+                workerState.checkForUpdatedBlobsTimer.scheduleRecurring(0,
```

Comment:
OK.  We have so many now it might be good to switch it over to a pool in the future, but nothing you need to worry about.